playbooks/roles/mongodb/files/logrotate

```diff
@@ -0,0 +1,8 @@
+/var/lib/mongodb/log/mongodb.log {
```

Comment:
Nit: this should be configurable and then become a Jinja2 template

Reply:
I was following the way we do it for prometheus. Will fix.

Reply:
@rocioar For Prometheus, I didn't make the log file location configurable at all.  In this case however, the location is already configurable, so it makes sense to use a template here.